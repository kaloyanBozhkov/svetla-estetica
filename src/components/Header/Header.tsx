import type { ReactElement } from 'react'
import styles from './header.module.scss'

const Header = (): ReactElement => (
    <div className={styles.header}>
        <div className={styles.logo}>
            <p>Svetla Estetica</p>
            <a href="#google">Dalmine (BG) - Viale Natale Betelli, 51</a>
        </div>
        <div className={styles.info}>
            <a href="tel:+393935026350">(+39) 393 5026 350</a>
            <p>
                <span>Luv - Ven:</span>
                <span>9.00 - 20.00</span>
            </p>
            <p>
                <span>Sabato:</span>
                <span>9.00 - 18.00</span>
            </p>
        </div>
    </div>
)

export default Header
