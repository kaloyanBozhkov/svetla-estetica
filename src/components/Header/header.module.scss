@import "scss/variables.scss";
@import "scss/mixins.scss";
@import "scss/breakpoints.scss";

.header {
    height: 100%;
    width: 100%;
    background-color: $white;
    display: flex;
    justify-content: space-between;
    align-items: center;
    position: relative;
    z-index: 10;

    @include sidePaddings();
    @include slyBoxShadow();

    .logo {
        display: flex;
        flex-direction: column;
        justify-content: center;
        p {
            font-size: 3rem;
            color: $black;
            margin-bottom: 8px;
        }

        a {
            font-size: 2rem;
            color: $grey;
        }
    }

    .info {
        display: flex;
        flex-direction: column;
        align-items: stretch;
        width: 20rem;

        a,
        p {
            font-size: 1.4rem;
            line-height: 2rem;
        }

        a {
            color: $black;
            font-weight: bold;
            text-align: right;
            margin-bottom: 8px;
            align-self: flex-end;
        }

        p {
            font-weight: lighter;
            display: flex;
            justify-content: space-between;
            align-items: center;
        }

        @include breakpoint(tablet-portrait) {
            display: none;
        }
    }
}
