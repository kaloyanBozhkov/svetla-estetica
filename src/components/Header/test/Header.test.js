import { shallow } from 'enzyme'
import React from 'react'
import Header from '../Header'

describe('testing Header', () => {
    it('should render component', () => {
        expect(shallow(<Header />)).toMatchSnapshot()
    })
})
