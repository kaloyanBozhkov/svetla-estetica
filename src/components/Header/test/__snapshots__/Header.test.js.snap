// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Header should render component 1`] = `
<div
  className="header"
>
  <div
    className="logo"
  >
    <p>
      Svetla Estetica
    </p>
    <a
      href="#google"
    >
      Dalmine (BG) - Viale Natale Betelli, 51
    </a>
  </div>
  <div
    className="info"
  >
    <a
      href="tel:+393935026350"
    >
      (+39) 393 5026 350
    </a>
    <p>
      <span>
        Luv - Ven:
      </span>
      <span>
        9.00 - 20.00
      </span>
    </p>
    <p>
      <span>
        Sabato:
      </span>
      <span>
        9.00 - 18.00
      </span>
    </p>
  </div>
</div>
`;
